kili/mutations/issue/__init__.py

```diff
@@ -0,0 +1,63 @@
+"""
+Issue mutations
+"""
+
+from typing import Optional
+from typeguard import typechecked
+
+from ...helpers import Compatible, format_result
+from .queries import GQL_APPEND_TO_ISSUES
+
+
+class MutationsIssue:
+    """Set of Issue mutations."""
+
+    # pylint: disable=too-few-public-methods,too-many-arguments,too-many-locals
+
+    def __init__(self, auth):
+        """Initialize the subclass.
+
+        Args:
+            auth: KiliAuth object
+        """
+        self.auth = auth
+
+    @Compatible(["v1", "v2"])
+    @typechecked
+    def append_to_issues(
```

Comment:
@PierreLeveau All mutations should be by batch to uniformise all methods, what do you think ?

Reply:
agreed

Reply:
Cannot append in batch because the resolver does not support it